### PR TITLE
Stop running 0th timestep

### PR DESCRIPTION
**Pull request by olyson**
Changes needed to stop running 0th timestep.
See CTSM issue https://github.com/ESCOMP/CTSM/issues/925 and PR https://github.com/ESCOMP/CTSM/pull/2084

### CODE REVIEW

File: src/riverroute/RtmHistFile.F90 [hunk 1]
```diff
@@ -983,8 +983,9 @@ subroutine RtmHistHtapesWrapup( rstwr, nlend )
     ! and write data to history files if end of history interval.
     do t = 1, ntapes
 
-       ! Skip nstep=0 if monthly average
-       if (nstep==0 .and. tape(t)%nhtfrq==0) cycle
+!KO    ! Should no longer be needed
```
Comment (jedwards4b) [inline src/riverroute/RtmHistFile.F90:986]: No need to leave commented code here.  Please remove.

File: src/riverroute/RtmTimeManager.F90 [hunk 2]
```diff
@@ -1000,7 +1000,10 @@ logical function is_first_step()
    call ESMF_ClockGet( tm_clock, advanceCount=step_no, rc=rc )
    call chkrc(rc, sub//': error return from ESMF_ClockGet')
    nstep = step_no
-   is_first_step = (nstep == 0)
+!KO   is_first_step = (nstep == 0)
```
Comment (jedwards4b) [inline src/riverroute/RtmTimeManager.F90:1003]: Remove commented code.

**Review by jedwards4b — CHANGES_REQUESTED** [review]
Thank you for the change @olyson.  Please remove the comments.  They are forever preserved in github.

File: src/riverroute/RtmHistFile.F90 (hunk 1 above)
Comment (olyson) [reply src/riverroute/RtmHistFile.F90]: Done.

File: src/riverroute/RtmTimeManager.F90 (hunk 2 above)
Comment (olyson) [reply src/riverroute/RtmTimeManager.F90]: Done

**Review by jedwards4b — APPROVED** [review]

**Comment by slevis-lmwg** [comment]
@olyson I assigned you because you worked on this, not because i need something from you at this time.

**Comment by slevis-lmwg** [comment]
@olyson I have @jedwards4b approval here, but mosart's timemanager changed since you worked on this, such that the line
`is_first_step = (nstep == 0)`
is not there to remove anymore. From a quick look, I don't see other cases of nstep that need modification, but could you confirm?

**Comment by slevis-lmwg** [comment]
derecho IN PROGRESS
`./run_sys_tests -s mosart -c mosart1.1.04-ctsm5.2.009 -g mosart1.1.05-ctsm5.2.011`

izumi

**Pull request merged**